src/foam/nanos/crunch/ServerCrunchService.java

```diff
@@ -51,10 +59,12 @@ public List getCapabilityPathFor(X x, String rootId, boolean filterGrantedUCJ, U
       requestedSubject.setUser(effectiveUser);
       x = x.put("subject", requestedSubject);
     }
-    return this.getCapabilityPath(x, rootId, filterGrantedUCJ);
+    return this.getCapabilityPath(x, rootId, filterGrantedUCJ, true);
   }
 
-  public List getCapabilityPath(X x, String rootId, boolean filterGrantedUCJ) {
+  public List getCapabilityPath(
+    X x, String rootId, boolean filterGrantedUCJ, boolean groupPrereqAwares
```

Comment:
Should  we have  left `public List getCapabilityPath(X x, String rootId, boolean filterGrantedUCJ)`  which calls 

`public List getCapabilityPath(    X x, String rootId, boolean filterGrantedUCJ, boolean groupPrereqAwares )` but  uses a default value  for groupPrereqAwares. 

Just  wondering if it makes sense in the rest of the codebase to always need to pass in a value for groupPrereqAwares, as it seems like  this only needed  to be added  because of this use case.

Reply:
Overloaded methods don't work so well with FOAM interfaces